gradle/validation/error-prone.gradle

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+configure([project(":lucene:core")]) {
+    apply plugin: 'net.ltgt.errorprone'
+
+    plugins.withType(JavaPlugin) {
+        dependencies {
+            errorprone("com.google.errorprone:error_prone_core:2.4.0")
+        }
+
+        tasks.withType(JavaCompile) {
+            options.errorprone.errorproneArgs = ['-Xep:InvalidBlockTag:OFF', '-Xep:MissingSummary:OFF', '-Xep:UnusedVariable:OFF']
```

Comment:
The gradle plugin has a nice DSL for setting options, see here?
https://github.com/tbroyer/gradle-errorprone-plugin#migration-from-versions-00x